frappe/oauth.py

```diff
@@ -575,7 +575,7 @@ def get_userinfo(user):
 		if frappe.utils.validate_url(user.user_image, valid_schemes=valid_url_schemes):
 			picture = user.user_image
 		else:
-			picture = frappe_server_url + "/" + user.user_image
+			picture = frappe_server_url + user.user_image
```

Comment:
This will break when user forgets to add the `/` in URL. 


Use `urljoin` instead.

Reply:
Also extra `/` usually dont break things afaik.

Reply:
Let me check..

Reply:
> This will break when user forgets to add the `/` in URL. 
> 
> 
> 
> 
> 

Actually, it's not coming from URL, the image url is already prefixed with a "/"

Reply:
`urljoin` handles all the scenarios. Yeah, it doesn't break anything, but still, replaced with `urljoin`.